contrib/netperf/collectd-td.spec.in

```diff
@@ -1833,8 +1836,16 @@ if [ $1 -eq 2 ]; then
 fi
 %systemd_post collectd.service
 %else
-/sbin/chkconfig --add cgconfig || :
-/sbin/chkconfig --add collectd-td || :
+/sbin/chkconfig --add cgconfig ||
+    echo 2>&1 '"chkconfig --add cgconfig" failed (keep going)'
+/sbin/chkconfig --add collectd-td ||
+    echo 2>&1 '"chkconfig --add collectd" failed (keep going)'
+levels=`/bin/sed -r -n 's/^ *# *chkconfig: *([0-9]+).*$/\1/p' \
```

Comment:
Before it added both services. BUT!  `cgconfig` is configured to run by default on *NO* run levels.

The other thing this diff does is to add a dependency to ensure chkconfig will order them correctly. That's the non-chkconfig syntax. The old chkconfig syntax still puts them in the right order (5 and 10), but that's much more fragile than directly specifying the order. Note that ordering and actually running the service are two completely different matters :-/